lib/cn/kvset_builder.c

```diff
@@ -423,6 +423,12 @@ kvset_builder_finish(struct kvset_builder *imp)
          */
         vbb_destroy(imp->vbb);
         imp->vbb = NULL;
+
+        if (adopted_vbs) {
+            blk_list_free(&imp->vblk_list);
```

Comment:
Oh nice, is this the memory leak fix? Good catch

Reply:
Is there an easy way to test this?

Reply:
Yes, the changes in my last commit is the fix for NFSE-5355. It takes 10 min. or so to repro this with the command line that I had posted in this ticket.